clients/cobol-lsp-vscode-extension/src/services/CopybooksDownloader.ts

```diff
@@ -110,7 +110,7 @@ export class CopybooksDownloader implements vscode.Disposable {
                         }
                     }
                     if (this.queue.length === 0 && errors.size > 0) {
-                        this.resolver.processDownloadError("Can't download copybooks: " + Array.from(errors));
+                        this.resolver.processDownloadError("Some copybooks could not be downloaded. Please provide correct settings for nested copybooks and fetch again. Missing copybooks: " + Array.from(errors));
```

Comment:
Same comment here, are you making sure that all the copybooks which are not downloaded are nested? if no, then the message should be rewritten

Reply:
Thanks, changed.